server/plugins/ssr_data.js

```diff
@@ -1,47 +1,53 @@
-var originalSubscribe = Meteor.subscribe;
-Meteor.subscribe = function(pubName) {
-  var params = Array.prototype.slice.call(arguments, 1);
+const originalSubscribe = Meteor.subscribe;
 
-  var ssrContext = FlowRouter.ssrContext.get();
-  if(ssrContext) {
-    FlowRouter.inSubscription.withValue(true, function() {
+Meteor.subscribe = function (pubName) {
```

Comment:
You can get params like this too: `pubName, ...params`

But, we need to use `arguments` below. So, I'm okay with this too.